Userland/Utilities/file.cpp

```diff
@@ -51,6 +52,39 @@ static Optional<DeprecatedString> image_details(StringView description, StringVi
     return builder.to_deprecated_string();
 }
 
+static Optional<DeprecatedString> audio_details(StringView description, StringView path)
+{
+// If serenity_main is defined as a macro, we're compiling for BuggieBox.
+#if defined(serenity_main)
+    return description_only(description, path);
+#else
+    auto loader_or_error = Audio::Loader::create(path);
+    if (loader_or_error.is_error())
+        return {};
+
+    auto loader = loader_or_error.release_value();
+    StringBuilder builder;
+    builder.appendff("{}, {} Hz, {}-bit {}, {} samples ({} s)\n",
+        description,
+        loader->sample_rate(),
+        loader->bits_per_sample(),
+        loader->num_channels() == 1 ? "Mono" : "Stereo",
+        loader->total_samples(),
+        loader->total_samples() / loader->sample_rate());
+    auto metadata = loader->metadata();
+    if (metadata.title.has_value())
+        // FIXME: Use “pretty quotation” once our terminal fonts support these characters.
+        builder.appendff("\"{}\"", metadata.title.value_or({}));
+    if (metadata.album.has_value())
+        builder.appendff(" (Album: {})", metadata.album.value_or({}));
+    // Use the quadruple ESC (a highly unlikely artist name) to detect when there's no artists.
+    if (auto all_artists = metadata.all_artists(String::from_utf8_short_string("\e\e\e\e"sv)); !all_artists.is_error() && all_artists.value() != "\e\e\e\e"sv)
```

Comment:
What? Needing \e\e\e\e makes this an incredible strange API. `ErrorOr<Optional<String>>`, returning nullopt when there are no artists, would be the idiomatic way of doing this.

Reply:
I disagree with your idea for all_artists, since that API intentionally returns an "unknown artist" text to be GUI-friendly (i.e. display code can call it without second thought and the user can always understand the result). However, the first_artist API (returning `Optional<Person>`) is the idiomatic way of checking this, since the internal logic for which person is an artist is identical to all_artists.